woocommerce/woocommerce-android
# TPP: Replace TTP specific illustration

Closes: #8144


### Description

The PR adds illustrations specific to the TPP flow

We have to use raster images, to keep shadows. I mitigated the size problem by converting them to webP.

More details on that - p1673265367766679-slack-C0354HSNUJH

### Testing instructions

* Run TPP flow in dark and light mode. Don't forget to simulate "error" state as well
* Notice the new illustrations

### Images/gif


https://user-images.githubusercontent.com/4923871/220044861-f4c8323e-8c33-4068-a2b5-57da84afb33d.mp4
